package.json

```diff
@@ -86,6 +123,7 @@
     "carbon-components": "10.12.0",
     "carbon-components-react": "7.12.0",
     "carbon-icons": "^7.0.7",
+    "chalk": "^4.1.0",
```

Comment:
Does this need to be a hard dependency? I think since it's just used in the formatter for development warnings, we can probably list it as a `devDependency` instead?